src/psyclone/parse/kernel.py

```diff
@@ -586,7 +586,7 @@ def get_kernel_metadata(name, ast):
     of the parse tree (a Fortran type) with the name 'name'.
 
     :param str name: the metadata name (of a Fortran type). Also \
-    the name referencing the kernel in the algorithm layer)
+    the name referencing the kernel in the algorithm layer). Case insensitive.
```

Comment:
Not your mistake originally but please could you remove the ")" after "algorithm layer".